src/app/@theme/components/language-switcher/language-switcher-list/language-switcher-list.component.ts

```diff
@@ -0,0 +1,53 @@
+import { Component, Input } from '@angular/core';
+import { NbPopoverDirective, NbLayoutDirection, NbLayoutDirectionService } from '@nebular/theme';
+import { TranslateService } from '@ngx-translate/core';
+
+@Component({
+  selector: 'ngx-lang-switcher-list',
+  templateUrl: './language-switcher-list.component.html',
+  styleUrls: ['./language-switcher-list.component.scss'],
+})
+
+export class LangSwitcherListComponent {
+
+  @Input() popover: NbPopoverDirective;
+  directions = NbLayoutDirection;
+  currentDirection: NbLayoutDirection;
+
+  languages = [];
```

Comment:
Might be worth adding typings to this variable

Reply:
let's do this

Reply:
done